app/controllers/administration_controller.rb

```diff
@@ -3,5 +3,13 @@
 class AdministrationController < ApplicationController
   # binding.pry
   before_action :authenticate_admin!
+  layout 'administration'
   def index; end
+
+  private
+
+  def authenticate_admin!
+    authenticate_user!
+    redirect_to root_url unless current_user.admin
+  end
```

Comment:
Doesn't need to create a custom method here, devise already handle dynamics helpers

Reply:
ok